.gitignore

```diff
@@ -0,0 +1,7 @@
+# OS files
+.DS_Store
+Thumbs.db
+
+# Local development
+.env
+.env.local
```

Comment:
What are we using these for? Are we using any .env or .env.local files in this repo? If not maybe remove then for now.